packages/node_modules/@ciscospark/plugin-messages/src/messages.js

```diff
@@ -74,11 +73,6 @@ const Messages = SparkPlugin.extend({
   onConversationActivityEvent(event) {
     const {activity} = event.data;
 
-    // Reply activities are not supported.
```

Comment:
How are we supporting replies now?

Reply:
We still aren't  Matt just convinced himself that we didn't need to have this line of code anymore.